packages/clay-css/src/scss/atlas/variables/_globals.scss

```diff
@@ -72,13 +72,42 @@ $teal: #20C997 !default;
 $cyan: #17A2B8 !default;
 
 $primary: #0B5FFF !default;
+$primary-d1: darken($primary, 5.10) !default;
+$primary-d2: darken($primary, 10) !default;
+$primary-l1: lighten($primary, 22.94) !default;
+$primary-l2: lighten($primary, 32.94) !default;
+$primary-l3: lighten($primary, 44.90) !default;
+
 $secondary: #6B6C7E !default;
-$success: #287D3D !default;
+$secondary-d1: #393A4A !default;
```

Comment:
Hey @pat270 we were wondering if we could base all $secondary variations in the main $secondary variable as you have done with $primary

If we cannot, maybe is better if we use the color, not the darken a lighten versions

Reply:
/cc @emiliano-cicero